openemory/publication/views.py

```diff
@@ -20,6 +20,7 @@
 import zipfile
 from slugify import slugify
 import time
+import magic
```

Comment:
I think this can be removed? Looks like you moved the mimetype function into utils.

Reply:
fixed